Sources/KVKCalendar/Timeline+Extension.swift

```diff
@@ -410,7 +410,7 @@ extension TimelineView {
     
     @objc func addNewEvent(gesture: UILongPressGestureRecognizer) {
         guard !isResizableEventEnable else { return }
-        
+        eventPreviewSize = getEventPreviewSize()
```

Comment:
Do we need to calculate size every time the gesture are triggered?

Reply:
Since `eventPreviewSize` is set to a new value every time `func didStartMovingEvent` is called and we can't tell if `eventPreviewSize` has been changed when `func addNewEvent` is called,  `eventPreviewSize`  has to be reset every time `func addNewEvent` is called.
If you don't want the size to be calculated every time `func addNewEvent` is triggered, the other way to solve this issue might be to separate the event preview size used in adding events and moving events.

Reply:
maybe we can move this line `eventPreviewSize = getEventPreviewSize()` inside `state = .began` to get the actual size only once 

<img width="602" alt="Screenshot 2024-03-27 at 2 50 13 PM" src="https://github.com/kvyatkovskys/KVKCalendar/assets/8233076/d576ec91-aab3-4c66-9089-51e4c6becb8e">

Reply:
Sure! I'll revise it later

Reply:
hmm... it must be updated before `eventPreviewSize` is used in
```
let newEventPreview = getEventView(style: style,
                                   event: newEvent,
                                   frame: CGRect(origin: point, size: eventPreviewSize))
```
So it doesn't work if it is moved inside the switch case.
How about calling it only when the state is `.began`?
```
  if gesture.state == .began {
      eventPreviewSize = getEventPreviewSize()
  }
```

Reply:
Looks good, thx 🙂